pkgs/top-level/all-packages.nix

```diff
@@ -11022,8 +11022,13 @@ let
 
   systemd = callPackage ../os-specific/linux/systemd {
     linuxHeaders = linuxHeaders_3_18;
+    cryptsetup = null; # Infinite recusion
   };
 
+  systemd-with-cryptsetup = appendToName "-with-cryptsetup" (systemd.override {
```

Comment:
I was under the impression that the convention favors dashes instead of underscores. I'm happy to change this if I'm mistaken.